torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -361,13 +361,15 @@ def inserting_before(self, node):
 
 
 def ReplaceMoreTtManually(gm: torch.fx.GraphModule, use_less_ttnn_op_types: bool) -> torch.fx.GraphModule:
+    again = False
     nodes = list(gm.graph.nodes)
     for node in nodes:
         if not can_lowering_to_ttnn(node):
             continue
         g = GraphWrapper(node)
 
         def rewrite_node(node):
+            nonlocal again
```

Comment:
Can you clarify what you achieve with this?

Reply:
the `aten.sum` will be lowered to `ttnn.sum` and `aten.squeeze`, so it need to run again to lower `aten.squeeze`. Currently rewrite_node's return is return node, but if let it return (node, again) it will change many code, so I use `nonlocal again` to set it

Reply:
I'm thinking if a simpler way to do this is "if rewrite_node() returns a new node -> graph_changed = True -> run ReplaceMoreTtManually again". WDYT?

This is less efficient (basically we always need to iterate twice) but I assume currrently the graph rewrite doesn't take too much time? To be more efficient, we can later track the new nodes created with `GraphWrapper.call_function` and only scan those nodes in the following iteration

Reply:
> I'm thinking if a simpler way to do this is "if rewrite_node() returns a new node -> graph_changed = True -> run ReplaceMoreTtManually again". WDYT?

ok, I change to use `modified`

> To be more efficient, we can later track the new nodes created with GraphWrapper.call_function and only scan those nodes in the following iteration

I'm not sure how's a proper way to know it is created with `GraphWrapper.call_function`

Reply:
And once applied `modified`, I think we can cancel the `decompose_aten_to_aten_ops`